common/util.go

```diff
@@ -497,3 +497,20 @@ func MimeTypeToExtension(mimeType string) (string, error) {
 	}
 	return "", ErrNoExtensionsForType
 }
+
+// ParsePricePerUnit parses a price string in the format <price><exponent><currency> and returns the price as a big.Rat and the currency.
+func ParsePricePerUnit(pricePerUnitStr string) (*big.Rat, string, error) {
+	pricePerUnitRex := regexp.MustCompile(`^(\d+(\.\d+)?([eE][+-]?\d+)?)([A-Za-z][A-Za-z0-9]*)?$`)
```

Comment:
Is this change backward-compatible? I see you use a different regex here, so I wonder, won't it break anything for the existing config?

Reply:
From a quick glimpse, it seems like the incompatibility would be if we had a currency being specified as `[eE][0-9]*` before, which will be detected as a scientific notation now. I think that case is pretty unlikely tho, but one solution could be to add a required space after the number in scientific notation (after `\d+`), which could be a good practice anyway for clearer `123e-12 USD` instead of a single `123e-12USD`. WDYT?

Reply:
Yeah, I'm fine even with some "slight" non-backward-compatibility as long as it's ok with Orchestrators.